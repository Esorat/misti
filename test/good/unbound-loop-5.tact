fun test(): Int {
  let a: Int = 10;
  repeat (a) {} // no warning, since the repeat is bounded by definition
  return a;
}
