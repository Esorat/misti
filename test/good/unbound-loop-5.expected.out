test/good/unbound-loop-5.tact:2:3:
  1 | fun test(): Int {
> 2 |   let a: Int = 10;
        ^
  3 |   repeat (a) {} // no warning, since the repeat is bounded by definition
Read-only variable
Help: Consider creating a constant instead
See: https://nowarp.github.io/docs/misti/docs/detectors/ReadOnlyVariables